packages/icon-button/index.js

```diff
@@ -0,0 +1,129 @@
+import React, {Component} from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import withRipple from '@material/react-ripple';
+import {MDCIconButtonToggleFoundation} from '@material/icon-button';
+
+const {strings} = MDCIconButtonToggleFoundation;
+
+export class IconButtonBase extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      classList: new Set(),
+      childContent: props.children,
+      [strings.ARIA_PRESSED]: props[strings.ARIA_PRESSED],
+      [strings.ARIA_LABEL]: props[strings.ARIA_LABEL],
+    };
+  }
+
+  componentDidMount() {
+    this.initializeFoundation();
+  }
+
+  initializeFoundation = () => {
```

Comment:
You only ever call this method from one place (line 22 above). So why have a separate method at all? Just put these two lines of code at line 22